man/paru.8

```diff
@@ -1,9 +1,9 @@
 '\ t
-.TH "PARU" "8" "2019\-10\-21" "paru v0.0.1" "Paru  Manual"
+.TH "PARU" "8" "2020\-10\-28" "paru v1.0.0" "Paru Manual"
 .nh
 .ad l
 .SH NAME
-PARU \- AUR Helper
+PARU \- An AUR helper and pacman wrapper
```

Comment:
Missed this last time but I think it's better without the `An`.

Reply:
I say AUR like I'm spelling it out, and an is the right word since A is a vowel sound. I can still change it if you like A better though.

Reply:
No I mean remove the world together `UR helper and pacman wrapper`. I believe that's the kind of language man pages tend to use. See man pacman/makepkg.

Reply:
Alright I brought that in line with how pacman and makepkg word theirs.